helm/examples/nfs/values-nfs.yaml

```diff
@@ -7,14 +7,52 @@
 # In this example the install name was "nfsstorage"
 
 storage:
+  planes:
+  - name: dali
+    pvc: dali-nfsstorage-hpcc-nfs-pvc
+    prefix: "/var/lib/HPCCSystems/dalistorage"
+    labels:
+    - dali
+
+  - name: dll
+    pvc: dll-nfsstorage-hpcc-nfs-pvc
+    prefix: "/var/lib/HPCCSystems/queries"
+    labels:
+    - dll
+
+  - name: sasha
+    pvc: sasha-nfsstorage-hpcc-nfs-pvc
+    prefix: "/var/lib/HPCCSystems/sasha"
+    labels:
+    - sasha
+
+  - name: data
+    pvc: data-nfsstorage-hpcc-nfs-pvc
+    prefix: "/var/lib/HPCCSystems/hpcc-data"
+    labels:
+    - data # NB: if not set, this would be the default
+
+
+
   dllStorage:
-    existingClaim: dll-nfsstorage-hpcc-nfs-pvc
+    plane: dll
     forcePermissions: true
 
   daliStorage:
-    existingClaim: dali-nfsstorage-hpcc-nfs-pvc
+    plane: dali
     forcePermissions: true
 
   dataStorage:
-    existingClaim: data-nfsstorage-hpcc-nfs-pvc
+    plane: data
     forcePermissions: true
+
+
+sasha:
+  wu-archiver:
+    storage:
+      plane: sasha
+      forcePermissions: true
+  dfuwu-archiver:
+    storage:
+      plane: sasha
+      forcePermissions: true
```

Comment:
NB: this is just an example, and it happens to use the plane name 'sasha' for both, but they could be different, or it could reuse one of the other planes.
Noting because I think there are issues around the forcePermissions code if not a single plane, or if prefix (defined in plane) is not what it expects.

Reply:
OK. Should I change anything here for now?